editors/vscode/esbuild.js

```diff
@@ -79,6 +79,6 @@ esbuild
         bundle: true,
         outfile: "out/propertiesView.js",
         platform: "browser",
-        format: "iife",
+        format: "esm",
```

Comment:
AFAICS this is used here:

```
			<body class="properties-editor">
                <script src="${scriptUri}"></script>
			</body>
```

The script tag also needs to be changed to `type = "module"`, no?

Reply:
Probably because the resulting bundled output file (`propertiesView.js`) doesn't end up using any import or export declarations.